test/tests/dgkernels/2d_diffusion_dg/tests

```diff
@@ -9,6 +9,15 @@
     requirement = "The system shall support solving 2D diffusion using the discontinuous Galerkin "
                   "method."
   []
+  [p_refinement_test]
+    type = 'Exodiff'
+    input = '2d_diffusion_dg_test.i'
+    cli_args = 'Outputs/file_base=p_refinement Executioner/Adaptivity/switch_h_to_p_refinement=true'
+    exodiff = 'p_refinement.e-s003'
+    max_parallel = 1
```

Comment:
This is somewhat worrisome.  Every libMesh `make check` does p-refinement examples in parallel; if we can't do the same in MOOSE too then there's probably some easy bug to fix.

Reply:
I've removed the restriction